src/effects/native/phasereffect.cpp

```diff
@@ -191,8 +191,12 @@ void PhaserEffect::processChannel(const ChannelHandle& handle,
         left = processSample(left, oldInLeft, oldOutLeft, filterCoefLeft, stages);
         right = processSample(right, oldInRight, oldOutRight, filterCoefRight, stages);
 
-        // Computing output combining the original and processed sample
-        pOutput[i] = pInput[i] + left * depth;
-        pOutput[i + 1] = pInput[i + 1] + right * depth;
+        // The depth knob acts like a dry/wet knob. The wet signal is made by adding
+        // the dry and computed sample, so to avoid making the output louder, divide
+        // the wet signal by 2.
+        pOutput[i] = pInput[i] * (1.0f - depth)
+                   + (pInput[i] + left) / 2.0f * depth;
```

Comment:
I think this works quite well except that it is difficult to hear before the depth knob is up half way. I think changing the curve of the parameter to some sort of logarithmic curve could solve that issue, but I don't quite understand the math behind ControlLogPotmeterBehavior or ControlAudioTaperPotBehavior. @daschuer do you have any ideas for how to improve the scaling?